sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure.AI.TextAnalytics.Models;
```

Comment:
StringIndexType is in the main namespace right? so this is no needed anymore?

Reply:
i ll remove it.